src/Datadog.Trace.ClrProfiler.Native/cor_profiler.h

```diff
@@ -62,6 +62,7 @@ class CorProfiler : public CorProfilerBase
     //
     // Helper methods
     //
+    WSTRING GetProfilerFilePath();
```

Comment:
Would it be safer to define this only for non-Windows builds? The symbol is still there for Windows builds so there's nothing stopping us from accidentally invoking this on Windows, and as it's written it may work incorrectly on Windows

Reply:
you're right, I think it will work on windows but only for .netcore (CORECLR env-vars)... I could make it work on .NET Framework too by doing a fallback to COR env-vars. The name of the function is descriptive and not related to any rewriting, just a get path... WDYT?

Reply:
Yeah I understand we could make it work with .NET Framework with the COR env-vars but I have no idea what happens if it's registered via COM. Since we don't need to get into that much detail and it's not necessary, I don't think we need to do Windows fallbacks right now. What if we just rename this to `GetCoreCLRProfilerPath`?

Reply:
done in b053eb8

Reply:
Thanks!